<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facial Database Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollBar" name="horizontalScroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>871</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(85, 170, 127);</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScroll">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>16</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(85, 170, 127);</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zoom">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>200</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="click_count_video">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>578</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00 Clicks</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_counting">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>600</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>620</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capture">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crop_image_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>250</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data_dir">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>360</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Data Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_dir">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>870</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="load_dir">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>230</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Load Data Directory</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFace">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Show Faces</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxEye">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Show Eyes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_img">
    <property name="geometry">
     <rect>
      <x>958</x>
      <y>270</y>
      <width>63</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_img">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>270</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="zoom_leval_display">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>605</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>1 X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cropall_image_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Crop All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loaded_label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>210</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(227, 227, 227);
color:rgb(255, 255, 255);
padding:2px;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Unload   x</string>
    </property>
   </widget>
   <zorder>horizontalScroll</zorder>
   <zorder>verticalScroll</zorder>
   <zorder>click_count_video</zorder>
   <zorder>image_counting</zorder>
   <zorder>file_name</zorder>
   <zorder>crop_image_btn</zorder>
   <zorder>data_dir</zorder>
   <zorder>output_dir</zorder>
   <zorder>display</zorder>
   <zorder>zoom</zorder>
   <zorder>load_dir</zorder>
   <zorder>checkBoxFace</zorder>
   <zorder>checkBoxEye</zorder>
   <zorder>next_img</zorder>
   <zorder>back_img</zorder>
   <zorder>zoom_leval_display</zorder>
   <zorder>cropall_image_btn</zorder>
   <zorder>capture</zorder>
   <zorder>loaded_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionNext_Image"/>
    <addaction name="actionBack_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuParameter">
    <property name="title">
     <string>Preference</string>
    </property>
    <widget class="QMenu" name="menuCapture">
     <property name="title">
      <string>Capture Ration</string>
     </property>
     <addaction name="actionOrignal_Size"/>
    </widget>
    <addaction name="actionSet_Lable"/>
    <addaction name="separator"/>
    <addaction name="menuCapture"/>
    <addaction name="separator"/>
    <addaction name="actionData_Directory"/>
    <addaction name="actionOutput_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionIP_Address"/>
    <addaction name="actionUser"/>
    <addaction name="actionPassword"/>
    <addaction name="actionDatabase_Name"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionUpload_To_Database"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCite"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentatin"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionCapture"/>
    <addaction name="separator"/>
    <addaction name="actionCrop_2"/>
    <addaction name="actionCrop_All_2"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Data"/>
    <addaction name="actionUnload_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuParameter"/>
   <addaction name="menuHelp"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSet_Lable">
   <property name="text">
    <string>Set Lable</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopyright_Notice">
   <property name="text">
    <string>Copyright Notice</string>
   </property>
  </action>
  <action name="actionCite">
   <property name="text">
    <string>Open GitHub</string>
   </property>
  </action>
  <action name="actionIP_Address">
   <property name="text">
    <string>IP Address</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionHost">
   <property name="text">
    <string>Host</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>User</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionPassword">
   <property name="text">
    <string>Password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDatabase_Name">
   <property name="text">
    <string>Database Name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionUpload_To_Database">
   <property name="text">
    <string>Upload To Database</string>
   </property>
  </action>
  <action name="actionOrignal_Size">
   <property name="text">
    <string>Orignal Size</string>
   </property>
  </action>
  <action name="actionData_Directory">
   <property name="text">
    <string>Data Directory</string>
   </property>
  </action>
  <action name="actionOutput_Directory">
   <property name="text">
    <string>Output Directory</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionDetect">
   <property name="text">
    <string>Detect</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionCapture_and_Detect">
   <property name="text">
    <string>Capture and Detect</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Camera</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close Camera</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionNext_Image">
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionBack_Image">
   <property name="text">
    <string>Back Image</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionCrop_All">
   <property name="text">
    <string>Crop All</string>
   </property>
  </action>
  <action name="actionLoad_Data">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="actionUnload_Data">
   <property name="text">
    <string>Unload Data</string>
   </property>
  </action>
  <action name="actionCrop_2">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionCrop_All_2">
   <property name="text">
    <string>Crop All</string>
   </property>
  </action>
  <action name="actionAssociated_People">
   <property name="text">
    <string>Associated People</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentatin">
   <property name="text">
    <string>Documentatin</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
